src/js/components/StatusBar.js

```diff
@@ -7,7 +7,7 @@ const GROW_RATIO = 0.07;
 // Grow anything below this threshold by stealing from the biggest portion
 const MIN_SIZE = 7; // (% of statusBar width)
 
-function stealPortion(barSizes, indexesLessThanThreshold) {
+function stealPortion(barSizes, indexesLessThanThreshold, unassignedPortion) {
```

Comment:
I think it would be premature to move into a Util because it's fairly specific to this Component.